includes/compatibility/edd/filters.php

```diff
@@ -30,11 +30,24 @@ function themedd_edd_settings_styles( $settings ) {
 	unset( $settings['main']['disable_styles'] );
 
 	// Remove "Default Button Color" option since Themedd controls all button styling
+	unset( $settings['button_text']['checkout_color'] );
 	unset( $settings['main']['checkout_color'] );
 
+	// Remove the button style selector.
+	unset( $settings['button_text']['button_style'] );
+
 	return $settings;
 }
-add_filter( 'edd_settings_styles', 'themedd_edd_settings_styles' );
+add_filter( 'edd_settings_misc', 'themedd_edd_settings_styles' );
+
+/**
+ * For EDD 2.x, use the `edd_settings_styles` filter to remove style settings.
+ *
+ * @return void
+ */
+if ( defined( 'EDD_VERSION' ) && version_compare( '2.10.999', EDD_VERSION, '>' ) ) {
```

Comment:
Fixing this.

Reply:
https://github.com/easydigitaldownloads/themedd/pull/256/commits/bc26a42a0a18902800b2341c0fb9e9a133631c4f